src/zypper_plugins/package_upload.py

```diff
@@ -51,13 +49,12 @@ def get_userdata(self, headers):
             logging.error("invalid userdata")
         return {}
 
-
-    def PLUGINBEGIN(self, headers, body):
+    def PLUGINBEGIN(self, headers, _body):
```

Comment:
If I'm very critical, this has the potential to be a breaking change. If it's called with keyword arguments (`x.PLUGINBEGIN(headers=headers, body=body`) then this can break because it's no longer exactly implementing the interface.

At the very least I wouldn't expect it in the same commit, but I'd suggest to leave it out altogether.

Reply:
Reverted `_` prefixed unused vars.